src/mmw/apps/export/views.py

```diff
@@ -75,12 +76,16 @@ def hydroshare(request):
     # POST existing resource updates it
     if hsresource and request.method == 'POST':
         # Update files
-        hs.add_files(hsresource.resource,
-                     params.get('files', []),
-                     overwrite=True)
-        hs.add_gms_files(hsresource.resource,
-                         params.get('mapshed_data', []),
-                         overwrite=True)
+        files = params.get('files', [])
+        for md in params.get('mapshed_data', []):
```

Comment:
Seeing an error for TR-55. I think we needed that guard for when there's no mapshed data:

```
TypeError at /export/hydroshare
'NoneType' object is not iterable
...
 126.     for md in params.get('mapshed_data', []):
```

Reply:
Or we could just send an empty array when not MapShed. Good catch, will fix.